ssl/ssl_local.h

```diff
@@ -1175,6 +1180,11 @@ struct ssl_ctx_st {
     uint32_t disabled_mkey_mask;
     uint32_t disabled_auth_mask;
 
+#ifndef OPENSSL_NO_QUIC
+    /* mapping from local IDs to QUIC packets to direct incoming packets correctly */
+    LHASH_OF(QUIC_ROUTE) *quic_route_table;
+#endif
```

Comment:
How then to figure out where incoming packets should be routed to?

The original design was for a promiscuous mode socket which would imply a global mapping.  @mattcaswell wanted this changed to be SSL_CTX based instead.  Can it live in the connection instead?  A lot has changed since the design was done (September give or take).